crates/meilisearch/tests/auth/authorization.rs

```diff
@@ -53,14 +53,14 @@ pub static AUTHORIZATIONS: Lazy<HashMap<(&'static str, &'static str), HashSet<&'
             ("PUT",     "/indexes/products/settings/sortable-attributes") =>   hashset!{"settings.update", "settings.*", "*"},
             ("PUT",     "/indexes/products/settings/stop-words") =>            hashset!{"settings.update", "settings.*", "*"},
             ("PUT",     "/indexes/products/settings/synonyms") =>              hashset!{"settings.update", "settings.*", "*"},
-            ("GET",     "/indexes/products/stats") =>                          hashset!{"stats.get", "stats.*", "*"},
-            ("GET",     "/stats") =>                                           hashset!{"stats.get", "stats.*", "*"},
-            ("POST",    "/dumps") =>                                           hashset!{"dumps.create", "dumps.*", "*"},
-            ("POST",    "/snapshots") =>                                       hashset!{"snapshots.create", "snapshots.*", "*"},
+            ("GET",     "/indexes/products/stats") =>                          hashset!{"stats.get", "*"},
+            ("GET",     "/stats") =>                                           hashset!{"stats.get", "*"},
+            ("POST",    "/dumps") =>                                           hashset!{"dumps.create", "*"},
+            ("POST",    "/snapshots") =>                                       hashset!{"snapshots.create", "*"},
             ("GET",     "/version") =>                                         hashset!{"version", "*"},
-            ("GET",     "/metrics") =>                                         hashset!{"metrics.get", "metrics.*", "*"},
-            ("POST",    "/logs/stream") =>                                     hashset!{"metrics.get", "metrics.*", "*"},
-            ("DELETE",  "/logs/stream") =>                                     hashset!{"metrics.get", "metrics.*", "*"},
+            ("GET",     "/metrics") =>                                         hashset!{"metrics.get", "*"},
+            ("POST",    "/logs/stream") =>                                     hashset!{"metrics.get", "*"},
+            ("DELETE",  "/logs/stream") =>                                     hashset!{"metrics.get", "*"},
```

Comment:
What does that change means? We cannot use the route with `stats.*` anymore?

Reply:
Hum... I think I understand, the `stats.*`, `dumps.*`, `snapshots.*` and `metrics.*` no longer exists as there is only a single variant for each of them. But is it something we want? Don't we want to keep them for backward compatibility?